src/Providers/ScoutApmServiceProvider.php

```diff
@@ -148,6 +164,17 @@ public function boot(
         $this->instrumentMiddleware($httpKernel);
     }
 
+    private function shouldInstrumentQueues(Config $config) : bool
```

Comment:
todo: verify this works in my test app.

Reply:
I tested a couple of scenarios to check everything works OK:

 * Added `$config['instrument_laravel_queues'] = 'false';` to `config/scout_apm.php`
 * Added `SCOUT_INSTRUMENT_LARAVEL_QUEUES=false` to `.env`

Both suppressed instrumenting queues :+1: